### ISSUE TITLE
Create Excel template export functionality 

### ISSUE THREAD

**Issue by tashapiro** [WA-Department-of-Agriculture/soil-health-report-generator#3]
Users need the ability to download the template file to fill out with their data as well as their corresponding data dictionary. File will be an Excel with two tabs - 1) for data and 2) for data dictionary.

**Comment by tashapiro** [comment]
@jadeynryan - will the Excel template be any different for Spanish reports? i.e. column names?

**Comment by jadeynryan** [comment]
Yes, the [data dictionary template](https://github.com/WA-Department-of-Agriculture/soils/blob/main/inst/templates/spanish/data/data-dictionary.csv) will be different for Spanish reports.

Like the English reports, the user should download the template with the two tabs then edit based on their own data.
